app/src/main/res/menu/menu.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Ignore this file, I thought that I had to implement in this part the navigation of the home screen...